src/test/java/nextstep/subway/AcceptanceTest/LineAcceptanceTest.java

```diff
@@ -0,0 +1,150 @@
+package nextstep.subway.AcceptanceTest;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.subway.api.LineApi;
+import nextstep.subway.api.StationApi;
+import nextstep.subway.domain.LineRepository;
+import nextstep.subway.domain.StationRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.jdbc.Sql;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Sql("classpath:/test-truncate.sql")
+@DisplayName("지하철 노선 관련 기능")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class LineAcceptanceTest {
+    private final LineApi lineApi = new LineApi();
+    private final StationApi stationApi = new StationApi();
+    static final String KEY_NAME = "name";
+
+    @LocalServerPort
+    int port;
+
+    @Autowired
+    StationRepository stationRepository;
+
+    @Autowired
+    LineRepository lineRepository;
+
+
+    Long upStationId1, upStationId2, downStationId1, downStationId2;
+    String lineName1, lineName2, lineColor1, lineColor2;
+
+    @BeforeEach
+    void setUp() {
+        if (RestAssured.port == RestAssured.UNDEFINED_PORT) {
+            RestAssured.port = port;
+        }
+        upStationId1 = stationApi.createId("광교역");
+        downStationId1 = stationApi.createId("신사역");
+        upStationId2 = stationApi.createId("석남역");
+        downStationId2 = stationApi.createId("장암역");
```

Comment:
테스트 픽스쳐를 고정된 값으로 넣고 있기에 가독성있게 변수명을 명명해도 괜찮을 것 같아요 😄 
그렇게 된다면 나 이외에 다른 직군분들과 같이 테스트할때도 이점이 있지 않을까 합니다!
물론 선택사항이오니 꼭 수정해주실 필요가 없습니다 :)
```suggestion
        광교역 = stationApi.createId("광교역");
        신사역 = stationApi.createId("신사역");
        석남역 = stationApi.createId("석남역");
        장암역 = stationApi.createId("장암역");
```